src/app-layout/__tests__/utils.tsx

```diff
@@ -39,11 +38,7 @@ export function renderComponent(jsx: React.ReactElement) {
   const isUsingGridLayout = wrapper.getElement().classList.contains(visualRefreshStyles.layout);
   const isUsingMobile = !!wrapper.findByClassName(testutilStyles['mobile-bar']);
 
-  const contentElement = isUsingGridLayout
-    ? wrapper.getElement()
-    : wrapper.findByClassName(styles['layout-wrapper'])!.getElement();
```

Comment:
This change unified the DOM structure between refresh and classic version, allowed to remove this condition